package.json

```diff
@@ -2,7 +2,7 @@
   "name": "tsd-jsdoc",
   "version": "2.0.0-beta.3",
   "description": "Compiles JSDoc annotated javascript into a Typescript Declaration File (.d.ts).",
-  "main": "publish.js",
+  "main": "dist/publish.js",
```

Comment:
@englercj I may make a mistake here. Not sure whether you release the dist dir of the repo itself.